movement/io/load_poses.py

```diff
@@ -313,27 +387,20 @@ def _from_lp_or_dlc_file(
 
     logger.debug(f"Loaded poses from {file.path} into a DataFrame.")
     # Convert the DataFrame to an xarray dataset
-    ds = from_dlc_df(df=df, fps=fps)
+    ds = from_dlc_df(df=df, fps=fps, source_software=source_software)
 
     # Add metadata as attrs
-    ds.attrs["source_software"] = source_software
     ds.attrs["source_file"] = file.path.as_posix()
 
-    # If source_software="LightningPose", we need to re-validate (because the
-    # validation call in from_dlc_df was run with source_software="DeepLabCut")
-    # This rerun enforces a single individual for LightningPose datasets.
-    if source_software == "LightningPose":
-        ds.move.validate()
-
     logger.info(f"Loaded pose tracks from {file.path}:")
     logger.info(ds)
     return ds
 
 
 def _load_from_sleap_analysis_file(
     file_path: Path, fps: Optional[float]
-) -> ValidPosesDataset:
-    """Load and validate data from a SLEAP analysis file.
+) -> xr.Dataset:
+    """Validate and load data from a SLEAP analysis file.
```

Comment:
I agree, but I wonder whether it's better to say "Create a movement dataset" instead of `xarray.Dataset`. 
After all we explicitly define what a "movement dataset" is [here](https://movement.neuroinformatics.dev/getting_started/movement_dataset.html).

Once we incorporate a sphinx glossary, it will be easy to link this term in the docstring it its glossary definition. What do you think?